pkg/executor/analyze.go

```diff
@@ -168,6 +169,10 @@ TASKLOOP:
 		}
 	}
 
+	failpoint.Inject("mockStuckAnalyze", func() {
+		time.Sleep(time.Second * 3)
+	})
```

Comment:
Looks like the stability of the test case relies on the timing.
After some thinking, I think we may be able to explicitly control how long it waits for in this way:
```go
if intest.InTest {
	for {
		stop := true
		failpoint.Inject("mockStuckAnalyze", func() {
			stop = false
		})
		if stop {
			break
		}
	}
}
```
Then it will keep waiting after we `Enable` the failpoint and before we `Disable` this failpoint.

Reply:
Updated.

Reply:
Thanks!